tests/test_stitch.py

```diff
@@ -1,8 +1,42 @@
-from morphocut.stitch import Region, Frame
+from morphocut.core import Pipeline, Call
+from morphocut.stitch import Region, Frame, Stitch
+from morphocut.stream import Unpack
 import numpy as np
 import pytest
 
 
+def test_Region():
+    region1 = Region(np.array([[1, 2], [3, 4]]), (slice(0, 2), slice(0, 2)))
+
+    region2 = Region(np.array([[1, 2], [3, 4]]), (slice(0, 2), slice(0, 2)))
+    region2[:, 1] = np.array([5, 6])
+    assert np.all(region2 == np.array([[1, 5], [3, 6]]))
```

Comment:
Instead of all `assert np.all(A == B)`, you should use `np.testing.assert_equal(A, B)` which produces better error messages.